tests/integration/helpers/project_builder.rs

```diff
@@ -95,6 +96,23 @@ impl ProjectBuilder {
             .success();
     }
 
+    fn git_commit(message: &str, current_directory: impl AsRef<std::path::Path>) {
+        Command::new("git")
+            .args([
+                "-c",
+                "user.name='Foo Bar'",
```

Comment:
great idea, to inline the git user detail here, normally on CI we set this extra up, but doing it this way would make CI simpler and we can reduce more boiler plate 👍

Reply:
Yeah, I was unsure how to run the tests locally too without adding this 😅 my git config picks user info depending if current directory is in my "personal" or "work" directory only.